src/data/todo.ts

```diff
@@ -25,7 +25,7 @@ export enum TodoSortMode {
 export interface TodoItem {
   uid: string;
   summary: string;
-  status: TodoItemStatus;
+  status?: TodoItemStatus;
```

Comment:
I think this is closer to the typing in core
```suggestion
  status: TodoItemStatus | null;
```